java/jar/jar.go

```diff
@@ -161,8 +161,7 @@ func extractManifest(ctx context.Context, name string, z *zip.Reader) (Info, err
 	return i, nil
 }
 
-// ExtractProperties pulls pom.properties files out of the META-INF directory
-// of the provided zip.
+// ExtractProperties pulls pom.properties files out of the provided zip.
 func extractProperties(ctx context.Context, name string, z *zip.Reader) ([]Info, error) {
 	const filename = "pom.properties"
 	mf, err := z.Open(`META-INF`)
```

Comment:
Removed the META-INF directory reference from the godoc because I don't see this variable actually used anywhere other than ensuring it exists? It doesn't look like we enforce pom.properties must exist within META-INF

Reply:
Originally, the package always used filesystem semantics (see c37acbc11d39b0ab08e9e489c9974c2c6b936d09), but this isn't quite right, so I expect this is a holdover comment.